agent/config/config.go

```diff
@@ -423,12 +426,23 @@ func getDockerStopTimeout() time.Duration {
 	parsedStopTimeout := parseEnvVariableDuration("ECS_CONTAINER_STOP_TIMEOUT")
 	if parsedStopTimeout >= minimumDockerStopTimeout {
 		dockerStopTimeout = parsedStopTimeout
-	} else if parsedStopTimeout != 0 {
+	} else {
 		seelog.Warnf("Discarded invalid value for docker stop timeout, parsed as: %v", parsedStopTimeout)
 	}
 	return dockerStopTimeout
 }
 
+func getContainerStartTimeout() time.Duration {
```

Comment:
It's still not clear to me what's happening in this method. If I specify the value as "-1m", this method returns `0`, correct? IMO, this method should never return a value that is less than `minimumContainerStartTimeout`. Can you please modify this to  be something like this: https://github.com/aws/amazon-ecs-agent/pull/1321#discussion_r177919115

Reply:
OK, I will make the change according to that comment.

Reply:
Update and add comments to getContainerStartTimeout and getDockerStopTimeout functions.